README.md

```diff
@@ -38,14 +38,6 @@ gitProperties {
 
 > Please note that `spring-boot-actuator` expects `git.properties` to be available at certain location.
 
-In some situations, for example when you run your build under a CI tool that checks out your code in detached mode, and HEAD no longer points to a branch name, but your CI tool
```

Comment:
@tha2015 why this README.md part was removed?

Reply:
It is removed because gitBranchName was removed from the config